beta-20170223.md

```diff
@@ -0,0 +1,62 @@
+---
+title: What&#39;s New in beta-20170223
+toc: false
+summary: Additions and changes in CockroachDB version beta-20170223.
+---
+
+## Feb 23, 2017
+
+Get future release notes emailed to you:
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170223.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170223.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### SQL Language Changes
+
+- Fixed parsing of interval constants with precision modifiers such as `INTERVAL '5' YEARS`. [#13702](https://github.com/cockroachdb/cockroach/pull/13702)
+- String literals containing `true`, `false`, or their synonyms (but not placeholders or other expressions of type string) can now be used where a `BOOL` is required. [#13685](https://github.com/cockroachdb/cockroach/pull/13685)
+- The underlying implementation of the `DECIMAL` type has been replaced. This changes the default rounding behavior of `DECIMAL` types, which is now consistent with PostgreSQL. [#13551](https://github.com/cockroachdb/cockroach/pull/13551)
```

Comment:
It may be good to note here that the results of some calculations may change, since that would surprise users.